File: LeetCode/1-100/26. 删除有序数组中的重复项.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int left = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i]> nums[left]) {
                nums[++left] = nums[i];
            }
        }
        return left + 1;
    }
}

File: LeetCode/1-100/26. 删除有序数组中的重复项.cpp
class Solution
{
public:
    int removeDuplicates(vector<int> &nums)
    {
        int l = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            if (nums[i] > nums[l])
            {
                nums[++l] = nums[i];
            }
        }
        return l + 1;
    }
};